meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -1,10 +1,24 @@
-# Support additional firmware for WiLink8 modules
+# Support additional firmware for WiLink8 and NXP Wireless modules
+#
 # TIInit_11.8.32.bts is required for bluetooth support but this particular
 # version is not available in the linux-firmware repository.
 #
+# sdiouart8987_combo_v0.bin is required for WiFi/BT support but this firmware
+# is not available in the linux-firmware repository.
+#
+# A code from meta-imx layer is used as a reference:
+# https://source.codeaurora.org/external/imx/meta-imx/tree/meta-bsp/recipes-kernel/linux-firmware/linux-firmware_%25.bbappend?h=hardknott-5.10.52-2.1.0
+#
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
 SRC_URI_append = "\
     https://git.ti.com/ti-bt/service-packs/blobs/raw/31a43dc1248a6c19bb886006f8c167e2fd21cb78/initscripts/TIInit_11.8.32.bts;name=TIInit_11.8.32 \
 "
+SRC_URI_append_imx = "\
+    git://github.com/NXP/imx-firmware.git;protocol=https;branch=lf-5.10.52_2.1.0;destsuffix=imx-firmware;name=imx-firmware \
```

Comment:
In case you change something on this commit, in future, maybe consider using a variable to set the branch value here, so when it is bumped we can change only the variable line. But this is not critical change, only a suggestion.

Reply:
Thanks, this is a draft so not late to change it before merging :) 'm fixing it in the next push.

Reply:
Fixed, thanks!